tier3/hooks/post_gen_project.py

```diff
@@ -0,0 +1,86 @@
+import subprocess
+
+REPO_NAME = '{{ cookiecutter.project_repo_name }}'
+ORG_NAME = '{{ cookiecutter.project_org }}'
+VISIBILITY = '{{cookiecutter.project_visibility}}'
+DESCRIPTION = '{{cookiecutter.project_description}}'
+CREATE_REPO = '{{cookiecutter.create_repo}}'
+RECEIVE_UPDATES = '{{cookiecutter.receive_updates}}'
+ADD_MAINTAINER = '{{cookiecutter.add_maintainer}}'
+
+def createGithubRepo():
+    subprocess.call(["git", "init", "-b", "main"])
+    subprocess.call(["git", "add", "."])
+    subprocess.call(["git", "commit", "-m", "first commit"])
+    gh_cli_command = [
+        "gh", "repo", "create",
+        f"{ORG_NAME}/{REPO_NAME}",
+        "--source=.",
+        f"--{VISIBILITY}",
+        "--push",
+        f"--description={DESCRIPTION}",
+    ]
+    subprocess.call(gh_cli_command)
+    subprocess.call(["git", "push", "--set-upstream", "origin", "main"])
+
+def addTopic():
+    gh_cli_command = [
+        "gh", "repo", "edit",
+        f"{ORG_NAME}/{REPO_NAME}",
+        "--add-topic=dsacms-tier0",
+    ]
+    subprocess.call(gh_cli_command)
+
+# Helper function for addMaintainer() to get user input of usernames for Maintainer, Approver, and Reviewer
+def get_usernames(role):
+    while True:
```

Comment:
Why is there less info asked for in the maintainers section in tier 3 than tier 2?

Reply:
Same thing with tier 4

Reply:
@IsaacMilarky I talked with @natalialuzuriaga and she said tier3/tier4 is a list of usernames for each role. I can ask Nat about it to double-check.

Reply:
That makes sense, resolving.